cygnss_wetlands/utils/constants.py

```diff
@@ -1,5 +1,7 @@
 CYGNSS_WAVELENGTH_M = 0.19  # meters
 
 EARTH_RADIUS_KM = 6371  # average radius of Earth, km
+CYGNSS_DISTANCE_TRAVELLED_IT_KM = 6  # km travelled by spacecraft during integration time
+CYGNSS_IDW_DISTANCE_THRESHOLD_KM = 8  # (for inverse distance weighting) distance threshold, km
```

Comment:
Any reason why?

Reply:
Ah-- now this is more arbitrary. I set this up to 8km given that theoretically the longest footprint could reach up to that length at certain look angles... but this is definitely a more gray decision.

Since not all footprints are that long .. and footprints are actually ellipses, not circular... so we are definitely over-estimating the footprints with this method and setting. My thinking was that with the inverse distance squared weights, very far points but still within this threshold are weighted much much less. But it does make for the case that this should only be used when aggregating a lot of data over a number of days (lets say at least 5 days+)

When looking at daily tracks, we should probably be looking at the footprint themselves and not relying on either of these gridded schemes